actionmailbox/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+*   Add ability to configure ActiveStorage service
+    for storing email raw source
```

Comment:
can you add an example here?

```suggestion
    for storing email raw source
    
    ```yml
    # config/storage.yml
    incoming_emails:
      service: Disk
      root: /secure/dir/for/emails/only
    ```
    
    ```ruby
    config.action_mailbox.storage_service = :incoming_emails
    ```
```

Reply:
(Minor sentence period to follow previous conventions.)

```suggestion
    for storing email raw source.
```